pkg/services/dashboards/service/metrics.go

```diff
@@ -0,0 +1,33 @@
+package service
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	metricsNamespace = "grafana"
+	metricsSubSystem = "dashboards"
+)
+
+func newDashboardsMetrics(r prometheus.Registerer) *dashboardsMetrics {
+	m := &dashboardsMetrics{
+		sharedWithMeFetchDashboardsSuccessRequestsDuration: prometheus.NewHistogram(
```

Comment:
do we need to distinguish between successful and failed requests as we do for the folders? if yes I suggest adding a `status` label.

Reply:
I would prefer to make a difference. So, yes, I can add labels to metric.